cdc/metrics_processor.go

```diff
@@ -75,28 +75,13 @@ var (
 			Help:      "The time it took to update sub changefeed info.",
 			Buckets:   prometheus.ExponentialBuckets(0.001 /* 1 ms */, 2, 18),
 		}, []string{"capture"})
-	tableOutputChanSizeGauge = prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Namespace: "ticdc",
-			Subsystem: "processor",
-			Name:      "txn_output_chan_size",
-			Help:      "size of row changed event output channel from table to processor",
-		}, []string{"changefeed", "capture"})
```

Comment:
Have you remove it from dashboard?

Reply:
yes, I renamed txn_output_chan_size to buffer_chan_size

Reply:
Does ticdc-grafana-dashboard.md reflect the change?

Reply:
yes